mysql-test/t/percona_innodb_fake_changes.test

```diff
@@ -368,5 +372,53 @@ COMMIT;
 
 DROP TABLE t1, t3;
 
+#-------------------------------------------------------------------------------
+#
+# Scenario will try to insert data that will cause split of a node something
+# which will not happen in real if fake_changes = 1.
+#
+
+# clustered index split
+use test;
+create table t1 (i int, j blob, primary key pk(i)) engine=innodb;
+insert into t1 values (1, repeat('a', 8000));
+insert into t1 values (2, repeat('b', 8000));
+set session innodb_fake_changes = 1;
+--error ER_ERROR_DURING_COMMIT
+insert into t1 values (3, repeat('c', 8000));
+set session innodb_fake_changes = 0;
+drop table t1;
+#
```

Comment:
I assume you are referring to empty lines or lines being with # but still empty.
I just used them to ensure proper formatting and it also helps in improving readability.

----

If you are referring to something else then probably I missed understanding the comment. Will sync with you over IM.